app/views/users/_user.html.erb

```diff
@@ -4,6 +4,10 @@
         <h2 class="user-name">
             <%=user.name%>
         </h2>
+        <% if user.id == current_user.id %>
+            <h2>CURRENT USER</h2>
+            <a href="/users/<%= user.id %>/posts/new" class="add-post-link">ADD NEW POST</a>
+        <% end %>
         <small class="user-posts-count">Posts: <%= user.posts_count || 0%></small>
```

Comment:
- [ ] The method `posts_count` does not exist which causes errors when running your project. You seem to have written it in multiple areas of this milestone which keeps on sending error messages. Please search for these methods and correct them.
![image](https://github.com/Tshobohwa/webwisesage/assets/45460192/fac93c9a-2ac3-481d-a8eb-67caca939f31)